### ISSUE TITLE
RC_PULS_REVERSE=1 doesn't seem to work after oneshot125 commit

### ISSUE THREAD

**Issue by fryguy1013** [sim-/tgy#122]
I was having problems getting RC_REVERSE_PULSE to function correctly. Upon looking through the code, I discovered that it seems to be caused by the oneshot125 commit (https://github.com/sim-/tgy/commit/6dbc056a71161593cc8eaf08f8959cf4ea6e4ce6). Going to the revision one before it allows RC_REVERSE_PULSE to work correctly.

**Comment by ahalekelly** [comment]
You mean RC_PULS_REVERSE? You had the order flipped, and I'm not sure if there's a typo in the name, or if there's a character limit, but there's no E in pulse.

**Comment by fryguy1013** [comment]
Yes, that one. I changed it by finding it in tgy.asm and changing the 0 to a 1, and the title here is from me typing it again from my head.
